### PR TITLE
chore: pre-release preparation

### PR DESCRIPTION
**Pull request by MelvinKl**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,4 +13,7 @@ lint:
 	# lint examples
 	flake8 --toml-config $(word 1, $(wildcard $(SERVICES_DIR)/*))/pyproject.toml --black-config $(word 1, $(wildcard $(SERVICES_DIR)/*))/pyproject.toml examples; 
 	# lint services
-	@for f in $(shell ls ${SERVICES_DIR}); do set -e; cd ${SERVICES_DIR}/$${f};flake8 .; cd ../..; done
\ No newline at end of file
+	@for f in $(shell ls ${SERVICES_DIR}); do set -e; cd ${SERVICES_DIR}/$${f};flake8 .; cd ../..; done
+
+lock:
+	@for f in $(shell ls ${SERVICES_DIR}); do set -e; cd ${SERVICES_DIR}/$${f};poetry lock --no-update; cd ../..; done
```
Comment (joaopalet) [inline Makefile:19]: When will this be used? And what is the goal?

Comment (MelvinKl) [reply Makefile:19]: It is used to update the dependencies in the lockfile. It needs to be used whenever the dependencies change, e.g. a new version of the `stackit_core`

Comment (joaopalet) [reply Makefile:19]: So probably something to be triggered automatically by Renovate or something, right?

**Review by joaopalet — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (MelvinKl) [reply Makefile:19]: Yes

**Pull request merged**